# Sometimes the trans directory isn't created before 
# accessing it.
# Upstream commit b452d84d2544a38eca1d2e4e9686a8eaef197d0f
# fixes this.
--- a/Makefile
+++ b/Makefile
@@ -535,6 +535,7 @@ endif
 # Build the translation files for gettext
 
 $(MOFILES): trans/%.mo: src/po/%.po
+	@-mkdir -p trans
 	msgfmt -o $@ $<
 
 %.desktop: %.desktop.in $(POTFILES)
@@ -552,14 +553,9 @@ translations: trans
 	@echo "Install gettext to run Tux Paint in non-U.S. English modes."
 	@echo "--------------------------------------------------------------"
 else
-translations: trans $(MOFILES) src/tuxpaint.desktop src/tuxpaint-fullscreen.desktop src/org.tuxpaint.Tuxpaint.appdata.xml
+translations: $(MOFILES) src/tuxpaint.desktop src/tuxpaint-fullscreen.desktop src/org.tuxpaint.Tuxpaint.appdata.xml
 endif
 
-trans:
-	@echo
-	@echo "...Preparing translation files..."
-	@mkdir trans
-
 ######
 
 windows_ARCH_INSTALL:=
